chromium/utils/translate-en-messages.py

```diff
@@ -1,139 +1,119 @@
+    'eu', 'fa', 'fi', 'fo', 'fr', 'gd', 'gl', 'gu', 'haw', 'he', 'hi', 'hr', 
+    'ht', 'hu', 'hy', 'id', 'is', 'it', 'ja', 'ka', 'kab', 'kk', 'km', 'kn', 
+    'ko', 'ku', 'ky', 'la', 'lb', 'lo', 'lt', 'lv', 'mg', 'mi', 'mk', 'ml', 
+    'mn', 'ms', 'mt', 'my', 'ne', 'nl', 'no', 'ny', 'pa', 'pap', 'pl', 'ps', 
+    'pt', 'ro', 'ru', 'rw', 'sg', 'si', 'sk', 'sl', 'sm', 'sn', 'so', 'sr', 
+    'sv', 'sw', 'ta', 'te', 'tg', 'th', 'ti', 'tk', 'tn', 'to', 'tpi', 'tr', 
+    'uk', 'ur', 'uz', 'vi', 'xh', 'yi', 'zh', 'zh-CN', 'zh-HK', 'zh-SG', 
+    'zh-TW', 'zu'
+]
 
-# UI initializations
-terminal_width = os.get_terminal_size()[0]
-def print_trunc(msg, end='\n') : print(msg if len(msg) < terminal_width else msg[0:terminal_width-4] + '...', end=end)
-def overwrite_print(msg) : stdout.write('\r' + msg.ljust(terminal_width)[:terminal_width])
+TERMINAL_WIDTH = os.get_terminal_size()[0]
 
-print('')
 
-# Prompt user for keys to ignore
+def print_trunc(msg, end='\n'):
+    """Prints a truncated message to fit terminal width."""
+    print(msg if len(msg) < TERMINAL_WIDTH else msg[:TERMINAL_WIDTH - 4] + '...', end=end)
+
+
+def overwrite_print(msg):
+    """Dynamically overwrites the current line in the terminal."""
+    stdout.write('\r' + msg.ljust(TERMINAL_WIDTH)[:TERMINAL_WIDTH])
+
+
+# Collect keys to ignore
 keys_to_ignore = []
 while True:
-    key = input('Enter key to ignore (or ENTER if done): ')
-    if not key : break
+    key = input('Enter key to ignore (or press ENTER if done): ')
+    if not key:
+        break
     keys_to_ignore.append(key)
 
-# Determine closest locales dir
-print_trunc(f'\nSearching for { locales_folder }...')
+# Locate locales directory
+print_trunc(f"\nSearching for {LOCALES_FOLDER}...")
 script_dir = os.path.abspath(os.path.dirname(__file__))
 locales_dir = None
-for root, dirs, files in os.walk(script_dir): # search script dir recursively
-    if locales_folder in dirs:
-        locales_dir = os.path.join(root, locales_folder) ; break
-else: # search script parent dirs recursively
-    parent_dir = os.path.dirname(script_dir)
-    while parent_dir and parent_dir != script_dir:
-        for root, dirs, files in os.walk(parent_dir):
-            if locales_folder in dirs:
-                locales_dir = os.path.join(root, locales_folder) ; break
-        if locales_dir : break
-        parent_dir = os.path.dirname(parent_dir)
-    else : locales_dir = None
-
-# Print result
-if locales_dir : print_trunc(f'_locales directory found!\n\n>> { locales_dir }\n')
-else : print_trunc(f'Unable to locate a { locales_folder } directory.') ; exit()
-
-# Load en/messages.json
+
+for root, dirs, _ in os.walk(script_dir):
+    if LOCALES_FOLDER in dirs:
+        locales_dir = os.path.join(root, LOCALES_FOLDER)
+        break
+
+if not locales_dir:
+    print_trunc(f"Unable to locate the {LOCALES_FOLDER} directory.")
+    exit()
+
+print_trunc(f"_locales directory found: {locales_dir}\n")
+
+# Load English messages
```

Comment:
Here your deletion of the `else` block means the script will no longer find locales folder if it exists in a parent dir